ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/platform.j2

```diff
@@ -20,7 +20,7 @@
 
 | Settings | Value |
 | -------- | ----- |
-{%             for sandsetting in platform.sand | arista.avd.natural_sort %}
+{%             for sandsetting in platform.sand | arista.avd.natural_sort if sandsetting not in ['qos_maps'] %}
```

Comment:
I had the same thought. But that would be something like below:

| Settings | Value |
| -------- | ------ |
| Internal Network QOS Mapping | TC0-to-QOS0, TC1-to-QOS7 |

Reply:
Sorry I was not clear. I meant for all the _other_ settings we should check for them one by one as usual, instead of iterating over the dict.
The table you built for the qos mapping is great.

Reply:
Got it.